renderers/SDL3/clay_renderer_SDL3.c

```diff
@@ -3,15 +3,18 @@
 #include <SDL3/SDL.h>
 #include <SDL3_ttf/SDL_ttf.h>
 
-/* This needs to be global because the "MeasureText" callback doesn't have a
- * user data parameter */
-static TTF_Font *gFonts[1];
```

Comment:
No longer needed (the function to measure the text gets it from the user data, and the rendering functions we have here get it from the `Clay_SDL3RendererData` struct pointer).